patches/minecraft/net/minecraft/server/level/ServerChunkCache.java.patch

```diff
@@ -24,11 +24,11 @@
 +   }
 +
 +   public <T> void registerTickingTicket(TicketType<T> type, ChunkPos pos, int distance, T value) {
-+      this.f_8327_.registerTicking(type, pos, distance, value);
++      this.f_8327_.addRegionTicket(type, pos, distance, value, true);
 +   }
 +
 +   public <T> void releaseTickingTicket(TicketType<T> type, ChunkPos pos, int distance, T value) {
-+      this.f_8327_.releaseTicking(type, pos, distance, value);
++      this.f_8327_.removeRegionTicket(type, pos, distance, value, true);
     }
```

Comment:
Since we're already doing breaking changes, might as well rename these methods to `addForcedTickingRegionTicket` and `removeForcedTickingRegionTicket` to bring them in line with the `addRegionTicket` and `removeRegionTicket` methods immediately preceeding these and the renames you did in `DistanceManager`, as well as communicating that these are region tickets which force ticking.

We could also rename these to exactly match `addRegionTicket` and `removeRegionTicket`, adding the `forceTicks` parameter at the end and using obfuscated parameters (with corresponding EXC entries).